class/data.php

```diff
@@ -0,0 +1,357 @@
+<?php
```

Comment:
I don't understand where this file is coming from or what it is used for. I don't recall that we ever worked on a core file like this, only Formulize files. In fact, the UCOSP branches don't include "core" files from outside modules/formulize. So what is this from?

Reply:
removed